localisation/atw_replace_l_english.yml

```diff
@@ -56,9 +56,6 @@
  despotism_desc:0 "Despotism is a form of government in which a single person rules with absolute power."
  oligarchism:0 "Oligarchic"
  oligarchism_desc:0 "Oligarchy is a form of government in which a group of people rule with absolute power."
- moderatism:1 "Moderatism"
```

Comment:
i prefer keeping this ideology.
naming however is `dem_moderatism` or `neu_moderatism`